src/main/java/ru/naumen/servacc/ui/UIController.java

```diff
@@ -257,25 +251,21 @@ public void widgetDefaultSelected(SelectionEvent selectionEvent)
         });
         toolitemCopy.addSelectionListener(new SelectionListener()
         {
-            @Override
             public void widgetDefaultSelected(SelectionEvent e)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)